libs/multiplayer/player.ts

```diff
@@ -438,6 +438,58 @@ namespace mp {
         player.setSprite(sprite);
     }
 
+    /**
+     * Selects one of the players by number
+     * @param number The player number
+     * @returns The player
+     */
+    //% blockId=mp_playerSelector
+    //% block="player $number"
+    //% group=Player
+    //% weight=100
+    //% blockGap=8
+    //% help=multiplayer/player-selector
+    //% parts="multiplayer"
+    export function playerSelector(number: PlayerNumber): Player {
+        const index = number - 1;
+        return getPlayerByIndex(index);
+    }
+
+    /**
+     * Returns an array of all players
+     */
+    //% blockId=mp_allPlayers
+    //% block="array of all players"
+    //% group=Player
+    //% weight=90
+    //% blockGap=8
+    //% help=multiplayer/get-all-players
+    //% parts="multiplayer"
+    export function allPlayers(): Player[] {
+        return _mpstate().players;
```

Comment:
maybe worth a slice to lessen the chances of anyone accidentally mutating it? I guess `_mpstate` is exported anyways so it'd just be to protect users in blocks from accidents~


```suggestion
        return _mpstate().players.slice();
```